src/interfaces.ts

```diff
@@ -476,11 +476,9 @@ export interface RunnerOptions extends WorkerPoolOptions {
 export interface CronJob {
   task: string;
   payload: {
-    _cron: { ts: string; backfilled?: boolean };
     [key: string]: unknown;
   };
   queueName?: string;
-  runAt: string;
```

Comment:
Why are we removing this? This definitely looks like a breaking change and I can't see the benefit?

Reply:
It's inserted in the DB query. No need to put the same information in multiple places IMO.